### DISCUSSION TITLE
What has happened to WSMP?

### DISCUSSION THREAD

**Discussion by nimgould** [coin-or/Ipopt#779] [General]
Dear IPOPTers

The links provided to the optional linear solver WSMP all lead to a 404 nowhere. Have IBM withdrawn the solver? That would be a shame. Or is this a temporary glitch?

Thanks in advance

Nick Gould, STFC-RAL

**Comment by svigerske** [comment]
I couldn't find an alternative page. So, we'll see if it comes back.

**Comment by nimgould** [comment]
Thank you. I wonder if anyone from IBM knows more? I gather that the author, Anshul Gupta, has retired. If so, is this the sort of thing that might be moved into coin-or?

**Reply by svigerske** [reply]
> I wonder if anyone from IBM knows more?

Me too. But also, who could this anyone be?

The initial idea of COIN-OR was not so much to dump code that noone wants to maintain anymore. But if IBM was interested to open-source WSMP, then that would be welcomed, I think.

**Comment by nimgould** [comment]
Yup, I appreciate that. It is bad enough trying to guess what is going on in SPRAL. As far as I recall, only the binaries were available for WSMP, so I have no idea what shape the code is in.
